roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -69,13 +69,15 @@ def run(self, tmp=None, task_vars=None):
                         msg=str(e),
                     )
 
+            if check.changed:
+                r["changed"] = True
```

Comment:
Pretty sure Ansible defaults to missing == False

For this code, it seemed simplest to always set it at the task level.